Adding copy script

jthermiz opened the pull request:
Added a script to strip the preprocessing and copy the stripped nwb to a new file. There is an option to overwrite the original file as well. This script is helpful because pynwb does not easily support overwriting nwb files.

JesseLivezey commented on scripts/copy_nwb_strip_processing.py:
```diff
@@ -0,0 +1,34 @@
+
+parser = argparse.ArgumentParser(description='Copies nwb and strips away processing module')
+parser.add_argument('src_file', type=str, help='Source nwb file')
+parser.add_argument('--dst_file', type=str, default=None,
```
I'm a bit wary of having the default option be to overwrite nwb files. IMO, users should have to specify that they want to overwrite things.

jthermiz commented:
> I'm a bit wary of having the default option be to overwrite nwb files. IMO, users should have to specify that they want to overwrite things. 

I changed the behavior so that the user has to provide the same destination file in order to overwrite the source file. 

I also changed the file name to be shorter and checked for flake8 errors
